recipes/nds2-client-swig/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nds2-client-swig" %}
+{% set version = "0.16.0" %}
+{% set sha256 = "db785868f519a07e3a1cf7216cf5610a984d96e3795862794de98215534aa463" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - pkg-config
+    - swig >=3.0.7
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - nds2-client ={{ version }}
+    - python
+    - numpy {{ numpy }}
+
+test:
+  imports:
+    - nds2
+
+outputs:
+  - name: python-nds2-client
+    requirements:
+      host:
+        - nds2-client ={{ version }}
```

Comment:
As ``nds2-client`` is a dependency and depends on ``python 2``, this recipe will not be able to create packages for python 3

Reply:
@marcelotrevisani, `nds2-client` only requires python for the build (specifically for ctest), and not for runtime.